trader.js

```diff
@@ -568,66 +975,116 @@ socket.on('close_traded_signal', async (signal) => {
             socket.emit("traded_buy_signal", traded_buy_signal)
             //////
             if (user_payload[tresult].trading_type === "real") {
-                console.log(signal.pair, ' ---==---> BUY ', signal.qty)
-                if (signal.pair == 'BTCUSDT') {
-                    bnb_client.mgMarketBuy("BTCUSDT", signal.qty, (error, response) => {
-                        if (error) console.log("ERROR 990099 BTCUSDT", Number(signal.qty), error.body)
-                        if (response) {
-                            console.log("----- mgRepay BTC -----")
-                            bnb_client.mgRepay("BTC", Number(signal.qty), (error, response) => {
-                                if (error) console.log("ERROR BTC 9", Number(signal.qty), error.body)
-                                else console.log("SUCCESS BTC 8")
-                            })
+                console.log(signal.pair, " ---==---> BUY ", signal.qty)
+                if (signal.pair == "BTCUSDT") {
+                    bnb_client.mgMarketBuy(
+                        "BTCUSDT",
+                        signal.qty,
+                        (error, response) => {
+                            if (error)
+                                console.log(
+                                    "ERROR 990099 BTCUSDT",
+                                    Number(signal.qty),
+                                    error.body
+                                )
+                            if (response) {
+                                console.log("----- mgRepay BTC -----")
+                                bnb_client.mgRepay(
+                                    "BTC",
+                                    Number(signal.qty),
+                                    (error, response) => {
+                                        if (error)
+                                            console.log(
+                                                "ERROR BTC 9",
+                                                Number(signal.qty),
+                                                error.body
+                                            )
+                                        else console.log("SUCCESS BTC 8")
+                                    }
+                                )
+                            }
                         }
-                    })
-                }
-                else {
-                    const alt = signal.pair.replace('BTC','')
-                    if (minimums[alt+'BTC'].minQty) {
-                        const qty = trading_qty[signal.pair+signal.stratid]
+                    )
```

Comment:
@JoeSchr your fork doesn't have issues enabled, so I'll want to let you now this way that your code on your fork's master has one curly brace too much at line 1012 :)

Reply:
Thanks!

Caught that on my end, but since there was no reaction I didn't bother pushing it again

Seems I should do that anyhow :)